pixray/Wallmaker_algorithm.md

```diff
@@ -0,0 +1,41 @@
+# Generate starting horizontal wall:
+
+Specify text `prompt`, `palette` (PNG file), output file `name`.
+Apply the following pixray settings:
+
+```python
```

Comment:
If it is intended to be one it's full of syntax errors.
Otherwise if it's just for syntax highlight, I get it.